tests/gui/builder/control/test_progress.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import taipy.gui.builder as tgb
+from taipy.gui import Gui
+from tests.gui.helpers import Helpers
+
+
+def test_progress_builder(gui: Gui, helpers: type[Helpers]):
```

Comment:
Why type[...] ?

Reply:
Actually that was a VSCode suggestion, I'll just remove it 👍